lib/fastly.js

```diff
@@ -138,8 +128,24 @@ function handleFastlyResponse(successMessage) {
     const options = {};
     console.log(prettyjson.render(fastlyResponseBody, options));
     if (successMessage) {
-      console.log(chalk.bold.green(successMessage));
+      console.log(chalk.green(successMessage));
     }
     process.stdin.pause();
   };
 }
+
+function fastlyApiKeyPresent(program) {
+  if (!program.apikey) {
+    console.error(chalk.red(fastlyApiKeyError));
+    return false;
+  }
+  return true;
+}
+
+function fastlyServiceIdPresent(program) {
```

Comment:
Only **named** functions are hoisted. When you assign an anonymous function as the value of a variable (whether it be an arrow function or with the `function` keyword), then it works like any variable. That is: it must be declared before it is used.

What @ryanhagerty suggests could still be accomplished if you make that `const` declaration higher in the file, before the function is invoked. Personally, I prefer using named functions declared near the bottom of a file, exactly as you have it here.